packages/utils/expand-transition.js

```diff
@@ -4,7 +4,9 @@ import WarpElement from '@warp-ds/elements-core';
 import { collapse, expand } from 'element-collapse';
 import { ifDefined } from 'lit/directives/if-defined.js';
 
-class ExpandTransition extends WarpElement {
+import { kebabCaseAttributes } from './index';
+
+class ExpandTransition extends kebabCaseAttributes(WarpElement) {
```

Comment:
We don't need it here either because only `show` property is connected to a `show` attribute. The private property `_removeElement` doesn't need to be mapped to an attribute.

Reply:
Updated in this commit: [9ddc7df](https://github.com/warp-ds/elements/pull/174/commits/9ddc7df079fcb7f4c661d5b5b6c0f8150dac7e89)